client/stylesheets/base.less

```diff
@@ -15,9 +15,9 @@
 	}
 }
 
-*,
-*:before,
-*:after {
+*:not(input),
+*:not(input):before,
+*:not(input):after {
 	.box-sizing(border-box);
```

Comment:
Can you maintain the box-sizing for all elements and just change this for user-select?